config/env_sample.hjson

```diff
@@ -57,6 +57,10 @@
     # SameSite settings for Session and CSRF (defaults in settings.py should work), if you do want non-string None set to null.
     "SESSION_COOKIE_SAMESITE": "None",
     "CSRF_COOKIE_SAMESITE": "None",
+    # user django session expiriration, defaults to 24hrs matching canvas
+    "SESSION_COOKIE_AGE": 60,
```

Comment:
Can we put here the default value of 86400 (aka 24hrs) here, which matches to the default value in line 137 of settings.py